### PR TITLE
Feature page skeleton

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

File: src/views/Account/index.tsx [hunk 1]
```diff
@@ -1,17 +1,61 @@
...
     const strings = useTranslation(i18n);
+    const { userDetails } = useContext(UserContext);
+    /*
+    const {
+        pending: mePending,
+        response: meResponse,
+    } = useRequest({
+        url: 'api/v2/user/me/',
+    });
+    */
```
Comment (AdityaKhatri) [inline src/views/Account/index.tsx:24]: we can remove this, right?

File: src/views/Country/index.tsx [hunk 2]
```diff
@@ -1,17 +1,205 @@
...
+                                className={styles.keyFigure}
+                                value={aggregatedAppealResponse.target_population}
+                                normalize
+                                description={strings.countryKeyFiguresTargetPop}
+                            />
+                            {countryResponse?.has_country_plan && (
+                                <KeyFigure
+                                    icon={<AppealsTwoIcon />}
+                                    className={styles.keyFigure}
+                                    value={1}
```
Comment (AdityaKhatri) [inline src/views/Country/index.tsx:142]: just to double check if this is dummy or correct

File: src/views/DrefApplicationForm/import.ts [hunk 3]
```diff
@@ -1,652 +0,0 @@
-import {
```
Comment (AdityaKhatri) [inline src/views/DrefApplicationForm/import.ts:1]: why is this removed?

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: src/views/Account/index.tsx (hunk 1 above)
Comment (frozenhelium) [reply src/views/Account/index.tsx:24]: We can move this to other component, where it is needed, but this is just skeleton for now.

File: src/views/Country/index.tsx (hunk 2 above)
Comment (frozenhelium) [reply src/views/Country/index.tsx:142]: This is correct, if there is a country plan, we show we have one country plan

File: src/views/DrefApplicationForm/import.ts (hunk 3 above)
Comment (frozenhelium) [reply src/views/DrefApplicationForm/import.ts:1]: It had tons of error which was blocking the build
